scss/_variables.scss

```diff
@@ -1267,3 +1267,7 @@ $kbd-color:                         $white !default;
 $kbd-bg:                            $gray-900 !default;
 
 $pre-color:                         null !default;
+
+// Position Sizes
+
+$position-sizes: () !default;
```

Comment:
We'll need documentation for those new utilities and this new Sass map, too.

Reply:
I'm going to work on that